CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to r7insight_ruby
+
+:+1::tada: Thanks for taking the time to contribute! :tada::+1:
+
+## Requirements
+
+In order to work on this repository you will require:
+- ruby version `2.6.5`
+- bundle version `2.0.2`
+- gem version `3.0.6`
+- rvm version `1.29.9`
+
+## Workflow
+
+- Fork repository in GitHub
```

Comment:
I would hope that anyone who wishes to make changes to this library at least knows how to use Git. I don't think we have to explain the whole of gitflow here ;)

Reply:
It's nice and explicit and allows people to hit the ground running quicker. Not like this can hurt.